doc/design/components.md

```diff
@@ -0,0 +1,252 @@
+# vSphere Integrated Containers - Architecture and Design
+
+## Approach
+
+The devleopment approach taken for VIC is iterative refinement, both for the high level product and recursively throughout the lower level components that go into it. This, as with most Agile processes,is driven by a need to have a useful product in short timeframes, and to manage risk - whether risk related to larger delivery deadlines (VIC may be OSS but it's still a VMware product), changing technical landscapes, or the unknown. The day-by-day development process we follow is [documented in the contributor docs](../../CONTRIBUTING.md).
+
+The upshot of this approach is that many of these components are outline sketches at best as we start out, with specific areas being filled out over time. Avoiding any comment on business priorities, the _technical_ priorities for the project are:
+
+1. build an appliance foundation to permit secure orchestration of Virtual Infrastructure by regular users
+2. build primitves that support container style semantics, implemented with VMs
+2. present the Docker API as a language with which users can perform the orchestration they require
```

Comment:
Consider... present a Docker personality provides a Docker API endpoint that enables users to orchestrate the system described in the first two bullets using the Docker API.